Pipfile

```diff
@@ -14,6 +14,8 @@ matplotlib = "*"
 scipy = "*"
 jupyter = "*"
 sklearn = "*"
+pandas = "*"
```

Comment:
This should be updated when you update your master repository and rebase this branch :)

Reply:
"pandas" was already added, so this commit can be dropped